addons/website_event/i18n/website_event.pot

```diff
@@ -723,6 +723,11 @@ msgstr ""
 msgid "General Questions"
 msgstr ""
 
+#. module: website_event
+#: model_terms:ir.ui.view,arch_db:website_event.event_description_full
+msgid "Get directions"
+msgstr ""
+
```

Comment:
Looks like this commit is in duplicate (one for translation, one for fix)
Can be fixuped :)

Also would be nice to know what the svg represents from the commit message, rather than having to check it out